cli.js

```diff
@@ -24,15 +24,12 @@ module.exports = async () => {
     }catch(err){
         await fs.remove(tempDir);
         if(err.code === 'ETARGET'){
-            spinner.fail(chalk.red(`version '${err.wanted}' not found in npm registry\navailable versions:`));
-            console.log(err.versions.reverse().join(' | '));
-            process.exit(1);
-        }/* istanbul ignore next */else{
-            spinner.fail('Unexpected error');
-            console.error(err);
-            process.exit(-1);
-        };
-        return;
+            const msg = chalk.red(`version '${err.wanted}' not found in npm registry\navailable versions:\n`)
```

Comment:
```suggestion
            const msg = chalk.red(`version '${err.wanted}' not found in npm registry\unavailable versions:\n`)
```

Reply:
It's meant to be /n - new line
Available versions: ...list of versions

Reply:
Ah, I scanned it wrong. ignore me.